integration/test/metric/cpu.go

```diff
@@ -0,0 +1,68 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
+
+package metric
+
+import (
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+type CPUMetricValueFetcher struct {
+	baseMetricValueFetcher
+}
+
+func (f *CPUMetricValueFetcher) Fetch(namespace string, metricName string, stat Statistics) ([]float64, error) {
+	dimensions := f.getMetricSpecificDimensions()
+	values, err := f.fetch(namespace, dimensions, metricName, stat)
+	if err != nil {
+		log.Printf("Error while fetching metric value for %v: %v", metricName, err.Error())
+	}
```

Comment:
I moved this here because I hated how all those logging at callers gets distracting for readers. While I do generally hate it, I also know some people prefer to log in caller.

Reply:
I'm fine with leaving it then